### PR TITLE
New module login support with language keys

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
The login screen contains a coustom module "Need Help?".
To localise this module we hav a new module mod_loginsupport. 
It can replace the custom_module "need help?" 

### Testing Instructions
Install the new module via discover.

In the Module Manager:
- Deactivate the module "needHelp".
- Make a new module of type "login support Information"
- You can write a own title or use the parameter "auto_title"
- Choose Position sidebar.

Now the module should be visible as before. 

### Documentation Changes Required
yes. This module must be described

### Additional Information
If this module is accepted, the install/update sql files must be changed.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.mod_loginsupport.ini [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+MOD_LOGINSUPPORT="Login Support Information"
+MOD_LOGINSUPPORT_DOCUMENTATION="Joomla! Documentation"
+MOD_LOGINSUPPORT_FORUM="Joomla! Support forum"
+MOD_LOGINSUPPORT_HEADLINE="You can find help here:"
+MOD_LOGINSUPPORT_NEWS="Joomla! News"
+MOD_LOGINSUPPORT_TITLE="Need Support?"
+MOD_LOGINSUPPORT_XML_DESCRIPTION="This module displays some useful links to Joomla support sites on the login screen. Customisation is possible via language overrides or template override"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.mod_loginsupport.ini]: personally i wouldnt bother with the text beginning "customisation...." that is true for every module ;)

File: administrator/language/en-GB/en-GB.mod_loginsupport.sys.ini [hunk 2]
```diff
@@ -0,0 +1,9 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+MOD_LOGINSUPPORT_XML_DESCRIPTION="This module displays some useful links to Joomla support sites on the login screen. Customisation is possible via language overrides or template override"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.mod_loginsupport.sys.ini]: this should be the last line and see previous note

File: administrator/modules/mod_loginsupport/mod_loginsupport.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="module" version="3.1" client="administrator" method="upgrade">
+	<name>mod_loginsupport</name>
+	<author>Joomla! Project</author>
+	<creationDate>June 2019</creationDate>
+	<copyright>Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.</copyright>
+	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+	<authorEmail>admin@joomla.org</authorEmail>
+	<authorUrl>www.joomla.org</authorUrl>
+	<version>3.0.0</version>
```
Comment (brianteeman) [inline administrator/modules/mod_loginsupport/mod_loginsupport.xml]: 4.0.0 or deploy_version please

**Comment by brianteeman** [comment]
Tested it and apart from the minor suggestions above and the obvious sql creation this is perfect for me although i suspect someone will suggest the urls should not be hard coded and should be module options

thank you

**Review by bembelimen — APPROVED** [review]

**Pull request merged**